data/README.rst

```diff
@@ -0,0 +1,107 @@
+Software Directory README
+#########################
+
+As decided by the XSF Board, the Software Directory will need yearly renewal for software to not disappear from the listing on the website. To help software authors with this, scripts are provided which help you with creating a proper pull request.
```

Comment:
The text could mention more explicitly that you don't _have_ to use the tool, but can manually edit the JSON files too. Although using the tool will safeguard against formatting errors, a simple edit of the JSON files will be preferred by some, as you can do that without executing anything.